FAQ.md

```diff
@@ -0,0 +1,29 @@
+# Is Pulsar-Edit the same as Atom-Community?
+
+While the original team that was working on Atom-Community is now the team creating Pulsar, they are not the same.
+
+They are two seperate forks of Atom. They have seperate goals, seperate contributors, and seperate editors. 
+
+Now of course any open source developer knows that teamwork across organizations and repos is essential to succeed, so in that vain Pulsar-Edit uses contributions from Atom-Community, and hopefully in the future that reverses to let Atom-Community use the wins of Pulsar.
+
+So with the above said if you have questions about Pulsar, or for the team working on it, please contact use on our GitHub Organization, the affected Repo, or of course on our Discord Server.
+
+# Why Split with Atom-Community, isn't more effort on one project better than two?
+
+While more effort on a single project is correct, since we are all volunteers here effort on a project you beleive in is most important. 
```

Comment:
```suggestion
While more effort on a single project is correct, since we are all volunteers here effort on a project you believe in is most important. 
```